Package.swift

```diff
@@ -31,8 +41,11 @@ let coreFoundationBuildSettings: [CSetting] = [
     .define("DEPLOYMENT_ENABLE_LIBDISPATCH"),
     .define("DEPLOYMENT_RUNTIME_SWIFT"),
     .define("HAVE_STRUCT_TIMESPEC"),
-    .define("SWIFT_CORELIBS_FOUNDATION_HAS_THREADS"),
+    .define("SWIFT_CORELIBS_FOUNDATION_HAS_THREADS", .when(platforms: platformsWithThreads)),
```

Comment:
Can we make this change to the CMake files as well so that the CMake and SwiftPM builds are in-sync?

Reply:
Done